src/components/Form.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from 'react';
+
+const Form = (props) => {
+
+    const [selectedGenre, setSelectedGenre] = useState("placeholder");
```

Comment:
I think it maybe more efficient to just have "" rather than "placeholder" - but really clean and solid code :D